nextcloud_update.sh

```diff
@@ -309,7 +309,22 @@ fi
 # Update all Nextcloud apps
 if [ "${CURRENTVERSION%%.*}" -ge "15" ]
 then
-    occ_command app:update --all
+    occ_command app:update --all >> Updated_Apps
+fi
+
+# Check which apps got updated
+if [ -n $(occ_command app:update --all) ]
+then
+    UPDATED_APPS=$(sed '1~2d' Updated_apps | awk '{print $1}')
+    for apps in $UPDATED_APPS
+    do
+        CHANGE_FORMATTING+="$apps"
+    done
+    CHANGE_FORMATTING=${CHANGE_FORMATTING[*]}
+    CHANGE_FORMATTING=${CHANGE_FORMATTING// /", "}
```

Comment:
Please explain what this does exactly, maybe there's a better way.

Reply:
i will do it on the new algorithm